contents/overlay/drawers/navigation-drawer/index.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  House,
+  Search,
+  Bell,
+  Heart,
+  Diamond,
+  User,
+  Settings,
+  LogOut,
```

Comment:
This is how you import icons.

https://github.com/yamada-ui/yamada-components/issues/301

```suggestion
  HouseIcon,
  SearchIcon,
  BellIcon,
  HeartIcon,
  DiamondIcon,
  UserIcon,
  SettingsIcon,
  LogOutIcon,
```